README.MD

```diff
@@ -78,9 +78,82 @@ The above example will execute the scenario loop over three times, once for each
-This project contains two Sub projects which can be used to build the Toolchain FitNesse plugin and host an FitNesse instance
 
-in this the Toolchain FitNesse plugin is the java project and the Toolchain FitNesse plugin test contains an FitNesse instance.
+# Getting Started with Toolchain FitNesse Plugin
+This is meant for people who want to develop for the Toolchain FitNesse Plugin.
 
-the Toolchain FitNesse plugin package is not used for anything except containing the two subprojects.
+## Running the code 
+
+1. Put the correct Toolchain Plugin version in the pom.xml.
+2. Press on "Maven" on the right side.
+    2. Open the folder “Lifecycle”.
+    2. Select “clean” and “install”. 
+    2. Press on “Run Maven Build”.
+
+The Plugin required to be included in a project with the original FitNesse. 
+
+## Code Set Up 
+
+### README
+
+There is a README in both projects that shows all current features. This README needs to be updated when a new feature is developed.   
+
+### Java
+
+To overwrite or add new responders, symbols, decorators in the FitNesse Toolchain you need to adjust the PraegusPluginFeatureFactory class.
+
+#### Responders
+
+Responder is the operation that the FitNesse Toolchain can perform for you. 
+The Responder is indicated in the URL such as "localhost:9090/FrontPage?editPage" where "?editPage" is the operation.
+
+#### Symbols
+
+Symbols is a special type of syntax that the user can use in the edit page.
+The user can for example use "!VersionChecker" to add a version checker table on the current page. 
+[HSAC symbols](https://github.com/fhoeben/hsac-fitnesse-plugin/tree/master/src/main/java/nl/hsac/fitnesse/symbols) was used as a example for creating this wiki symbol. 
+
+### Unit testing
+
+The written Java code needs to be tested. 
+The Java testing framework "JUNIT" is used for unit testing the code. 
+The created unit tests are located in the "test/java/nl/praegus/fitnesse" folder.
+The tests are integrated with a automated build in Github. When creating a pull request to the Praegus.
+
+## Code Conventions
+### General
+#### Naming and notation
+
+- Naming
+    - Do not change the existing name from the FitNesse master.
+    
+    
+- Notation
+    - Use Snake casing when naming tests.
+    - Use Pascal casing when naming functions.
+    - Use Camel casing when naming variables.
+    
+#### Formating
+
+- Format button
+    - Using the format button, everyone will get their code to look well arranged.
+    
+    
+- Enters
+    - Don't add unnecessary enters, but place them in places to provide more a beter overall picture. Just like with each new CSS element and between functions.
+    
+    
+- Spaces
+    - Place spaces where necessary. For example do not write an if statement like this: “if(saveButton==true){}” but like this “if (saveButton == true) { }“. As another example, do not write a variable like this ”const saveButton=false;" but like this “const saveButton = false;”
```

Comment:
Hier ook, javavoorbeelden aub.

Reply:
We hebben de overbodige/incorrecte tekst er uit gehaald.